backend/src/main/resources/graphql/main.graphqls

```diff
@@ -506,9 +506,11 @@ type Mutation {
     @requiredPermissions(allOf: ["MANAGE_USERS"])
   setUserIsDeleted(id: ID!, deleted: Boolean!): User
     @requiredPermissions(allOf: ["MANAGE_USERS"])
+  @deprecated(reason: "Use reactivateAndResetPassword version")
```

Comment:
Is the operation reactivateUser getting deprecated because the new reactivateUserAndResetPassword will be used in the Org admin/ manage user page instead?. If that is the case, does that need to be documented or communicated somehow to the org admin users?

Reply:
ooo good catch Johanna! @jayna-SkylightDigital do you have a strong preference here? If not, might be good to just stick with the status quo and continue using the reactivate only flow for org admins since we don't have the research like we do with the support admins on this.

lmk!

Reply:
@fzhao99 No preference on my end -- continuing to use the reactivate only flow for org admins makes sense for now.